content-security-policy/embedded-enforcement/allow_csp_from-header.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Embedded Enforcement: Allow-CSP-From header.</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="../support/testharness-helper.sub.js"></script>
+</head>
+<body>
+  <script>
+    var tests = [
+      { "name": "Same origin iframes are always allowed.", 
+        "origin": Host.SAME_ORIGIN,
+        "csp": "style-src 'unsafe-inline'; script-src 'unsafe-inline';", 
+        "allow_csp_from": "",
+        "expected": IframeLoad.EXPECT_LOAD,
+        "blockedURI": null},
+      { "name": "Cross origin iframe without Allow-CSP-From header gets blocked.", 
+        "origin": Host.CROSS_ORIGIN,
+        "csp": "script-src 'unsafe-inline';", 
+        "allow_csp_from": "",
```

Comment:
Can you distinguish between an _empty_ `allow-csp-from` and response where the header isn't present? Perhaps `null` as opposed to `""`?